services/director-v2/src/simcore_service_director_v2/models/dynamic_services_scheduler.py

```diff
@@ -426,6 +438,16 @@ def get_proxy_endpoint(self) -> AnyHttpUrl:
         "If set to None, the current product is undefined. Mostly for backwards compatibility",
     )
 
+    @root_validator(pre=True)
+    @classmethod
+    def _callbacks_mapping_legacy_migration(cls, values):
+        warnings.warn(
+            "check notes for deprecation at https://github.com/ITISFoundation/osparc-simcore/issues/4745"
+        )
+        if "callbacks_mapping" not in values:
```

Comment:
why not providing a default value instead of that validator?

Reply:
this works as well.